src/Apps/Artist/routes.tsx

```diff
@@ -97,7 +98,7 @@ export const routes: RouteConfig[] = [
         const { artist } = props as any
 
         if (!artist) {
-          return null
+          return undefined
```

Comment:
Need to figure out how to address this more globally, but in order to trigger the top loading bar we need to return `undefined` rather than `null`